packages/components/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import { ReactNode } from 'react';
+import {
+  Box,
+  Link,
+  SimpleGrid,
+  Stack,
+  Text,
+  useColorModeValue,
+} from '@chakra-ui/react';
+
+import { Logo } from '@packages/components/icons';
+import { useI18n } from '@packages/features/i18n-context';
+
+const ListHeader = ({ children }: { children: ReactNode }) => {
+  return (
+    <Text fontWeight="500" fontSize="lg" mb={2}>
+      {children}
+    </Text>
+  );
+};
+
+export default function Footer() {
+  const bgColor = useColorModeValue('gray.200', 'gray.800');
+  const color = useColorModeValue('gray.700', 'gray.200');
+  const { t } = useI18n('footer');
+
+  const socialLinksData = [
+    { url: 'https://github.com/podcodar', name: 'Github' },
+    { url: 'https://www.linkedin.com/company/podcodar/', name: 'LinkedIn' },
+  ];
+  const supportLinksData = [
+    { url: '/terms-of-service', name: t(`terms`) },
+    { url: '/privacy-policy', name: t(`privacy`) },
+  ];
+  const socialLinks = socialLinksData.map((obj) => (
```

Comment:
É super recomendavel evitar nomes genericos, tipo obj. No js literalmente TUDO é um object.

Aqui cairia bem um deocmposing/destructuring

Reply:
O mesmo vale para `supportLinks`

Reply:
Esses caras tbm tem que ir pra fora da função

Reply:
o problema que existe aqui em levar isso pra fora da funcao eh que isso complico o uso da traducao

Reply:
separado. uma coisa q surgiu eh q agora existem 3 mini componentes que sao o HeaderTitle, SupportLinks e SocialLinks (que estao dentro do footerData). Da pra colocar eles em arquivos separados tambem mas acho que neste caso ficaria pior para a legibilidade do codigo. Porem eh bom ter isso em mente pra quando for a hora de escalar o componente